dpti/aitools/codebase/collect_py_files_context.py

```
@@ -0,0 +1,163 @@
+# %%
+
+import argparse
+import json
+from pathlib import Path
+from typing import Dict
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)